OpenRA.Mods.Cnc/UtilityCommands/ImportTSMapCommand.cs

```diff
@@ -565,5 +566,50 @@ static void ReadLighting(Map map, IniFile file)
 				})));
 			}
 		}
+
+		/// <summary>
+		/// Replaces destroyable cliff templates with ramps + targetable cliff actor
```

Comment:
This doesn't seem to add the targetable cliff actor or I am missing something.

Reply:
See PR description.  I didn't update the comment when I removed that.